Makefile

```diff
@@ -0,0 +1,53 @@
+# You will need to use GNU make
+
+INCLUDES := $(realpath $(CURDIR)/include)
+CFLAGS += -pedantic -Wall -Wextra -Wno-unused-but-set-variable \
+	-Wno-sign-compare -Wno-variadic-macros
+
+
+LIBOBJ := build/color.o build/image_aspect.o \
+	build/stb_image.o build/stb_image_resize.o \
+	build/stb_image_write.o build/stb_truetype.o
+
+SRCOBJ := build/asciify_base build/asciify_blocks build/asciify_charmap
+
+all: lib src
+
+clean:
+	rm $(LIBOBJ) $(SRCOBJ)
+
+lib: $(LIBOBJ) 
+
+src: $(SRCOBJ)
+
+build/color.o: lib/color.c
+	$(CC) $(CFLAGS) $^ -I$(INCLUDES) -c -o $@ 
+
+build/image_aspect.o: lib/image_aspect.c
```

Comment:
That's kinda repetitive operation, but I suppose it is to support not just GNU Make? I think there was something like `%.o: %.c` or something. Anyway if it works, it works

Reply:
I was thinking that it wouldn't work, but i was wrong so please wait for fixes.

Reply:
Yeah, and also there is `lib/commons.c` which looks like you forgot to add (or it wasn't existing at the time you made Makefile), so I'll add it manually for now